phoenix-core/src/main/java/org/apache/phoenix/util/UpgradeUtil.java

```diff
@@ -1418,6 +1420,14 @@ public static void syncTableAndIndexProperties(PhoenixConnection conn, Admin adm
                 continue;
             }
             if (table.getType() == PTableType.INDEX) {
+                if (table.getIndexType() == IndexType.GLOBAL &&
+                        !table.isTransactional() &&
+                        !table.isImmutableRows() &&
+                        !origTableDesc.hasCoprocessor(GlobalIndexChecker.class.getName())) {
+                    TableDescriptorBuilder builder = TableDescriptorBuilder.newBuilder(origTableDesc);
+                    builder.addCoprocessor(GlobalIndexChecker.class.getName(), null, QueryServicesOptions.DEFAULT_COPROCESSOR_PRIORITY - 1, null);
```

Comment:
Need to remove the old Indexer co-processor as well.

Also, as I have noted in the upgrade steps I believe this change should not be in the 4.14.2 release and putting it in another PR would make it easier.

Reply:
I have already removed this code in my private branch and will be removed in the next patch. I am waiting for the next code review comments to do that.

Reply:
Done